deploy/examples/import-external-cluster.sh

```diff
@@ -196,7 +196,7 @@ parameters:
   pool: $RBD_POOL_NAME
   dataPool: $RBD_METADATA_EC_POOL_NAME
   imageFormat: "2"
-  imageFeatures: layering
+  imageFeatures: layering,fast-diff,image-map,deep-flatten,exclusive-lock
```

Comment:
How about adding a env var to pass these extra imageFeatures. 
We don't need to add them always since they require kernel support ?

Reply:
That was a question I had -- does CSI use KRBD mounts, or librbd?

Reply:
krbd is used by default. To use librbd you could set [mounter: rbd-nbd](https://github.com/rook/rook/blob/master/deploy/examples/csi/rbd/storageclass.yaml#L78-L81) in the storage class. However, it's not recommended for production as explained in that link.

Reply:
Thanks.  I did find some references to librbd in the source but wasn't sure what actually got used in production.

My reasoning is that most people deploying `master` or the forthcoming 1.10 will be using a 5.4/equivalent or later kernel, though I may be mistaken.  5.4 came out at the end of 2019.  That said, who knows what's in any given RHEL/CentOS kernel?  And some people are stuck on older kernels for other reasons.

I've refactored this script, PTAL.

Reply:
It's amazing how long old kernels linger, but perhaps it is reasonable to assume at least 5.4 as the default. @Madhu-1 wdyt?

Reply:
I have seen people still using 3.10 or 4.17 in slack/github issues, assuming 5.4 as default might not be the right one.

Reply:
3.10 is RHEL/CentOS, AIUI that will have various newer code cherry-picked. 

I still think 5.4+ as a baseline is reasonable, given that I've documented the dependency and what to do if one is stuck with an older kernel.  There will be a long tail of stragglers *forever*, at some point we have to move on.